src/test/java/com/adyen/CheckoutTest.java

```diff
@@ -798,6 +798,37 @@ public void TestBilldeskOnlineDetails(){
                 + "}",jsonRequest );
     }
 
+    @Test
+    public void TestBacsDirectDebitDetails() {
+        BacsDirectDebitDetails bacsDirectDebitDetails = new BacsDirectDebitDetails();
+
+        PaymentsRequest paymentsRequest = createBacsDirectDebitDetails();
+        paymentsRequest.setPaymentMethod(bacsDirectDebitDetails);
+
+        String jsonRequest = PRETTY_PRINT_GSON.toJson(paymentsRequest);
+
+        jsonRequest += "";
```

Comment:
why this is needed?

Reply:
This is in fact not needed. It is being removed in the next commit. 
I hope the next commit can be made part of this same pull request ?